unittest/gunit/keyring/keys_container-t.cc

```diff
@@ -1936,14 +1936,14 @@ TEST_F(Keys_container_with_mocked_system_keys_container_test,
   IKeyring_io *keyring_io = new Buffered_file_io(logger);
   Mock_system_keys_container *system_keys_container =
       new Mock_system_keys_container;
+  keys_container->set_system_keys_container(system_keys_container);
```

Comment:
Looks like change for this line should be reverted. At the moment test fails, also there is ASAN error 

==436521==ERROR: AddressSanitizer: heap-use-after-free on address 0x613000005500 at pc 0x0000007188fa bp 0x7fff52e85bb0 sp 0x7fff52e85ba8
#2 0x62143d in keyring__keys_container_unittest::Keys_container_with_mocked_system_keys_container_test_ErrorFromRotateKeyWhenStoringKey_Test::TestBody() unittest/gunit/keyring/keys_container-t.cc:1943

freed by thread T0 here:
...
#9 0x62113f in keyring__keys_container_unittest::Keys_container_with_mocked_system_keys_container_test_ErrorFromRotateKeyWhenStoringKey_Test::TestBody() unittest/gunit/keyring/keys_container-t.cc:1940